packages/datatrak-web/src/features/Survey/SurveySuccessScreen.tsx

```diff
@@ -5,31 +5,86 @@
 
 import React from 'react';
 import styled from 'styled-components';
-import { Paper as MuiPaper } from '@material-ui/core';
+import { Typography } from '@material-ui/core';
+import { useForm } from 'react-hook-form';
+import { Button as BaseButton } from '../../components';
+import { generatePath, useNavigate, useParams } from 'react-router-dom';
+import { useSurveyForm } from './SurveyContext.tsx';
+import { ROUTES } from '../../constants';
 
-const Container = styled.div`
-  flex: 1;
+const Wrapper = styled.div`
   display: flex;
-  padding-top: 2rem;
-  padding-bottom: 2rem;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 2rem 1rem;
+  flex: 1;
+`;
+
+const StyledImg = styled.img`
+  max-width: 80%;
+  max-height: 50%;
+  margin-bottom: 2.75rem;
 `;
 
-const Paper = styled(MuiPaper).attrs({
-  variant: 'outlined',
-  elevation: 0,
+const Title = styled(Typography).attrs({
+  variant: 'h2',
 })`
-  flex: 1;
-  overflow: auto;
-  padding: 3rem;
-  max-width: 70rem;
-  margin: 0 auto;
+  font-size: 1.9rem;
+  font-weight: 600;
+  text-align: center;
+  margin-bottom: 1.19rem;
+`;
+
+const Text = styled(Typography)`
+  max-width: 34.6rem;
+  width: 100%;
+  text-align: center;
+  margin-bottom: 1.875rem;
 `;
+
+const ButtonGroup = styled.div`
+  max-width: 28rem;
+  width: 100%;
+`;
+
+const Button = styled(BaseButton)`
+  & + & {
+    margin: 1.25rem 0 0 0;
+  }
+`;
+
 export const SurveySuccessScreen = () => {
+  const params = useParams();
+  const navigate = useNavigate();
+  const { activeScreen, formData, setFormData } = useSurveyForm();
+  const { reset } = useForm({ defaultValues: formData });
+
+  const repeatSurvey = data => {
+    setFormData(reset({ ...formData, ...data }));
```

Comment:
This doesn't work. We don't need reset since that won't change the global form context value and setFormData wont work with empty values because of the way we spread the updated values in the form context reducer. I think it would be nice to add a new method to the reducer for resetting the form state and then we could use that here.